Description: Adds missing files.
 Note: upstream has already a fix here:
 https://review.openstack.org/#/c/123306/
 .
 So this patch is to be removed when the final Juno release is out.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-09-22

Index: sahara/MANIFEST.in
===================================================================
--- sahara.orig/MANIFEST.in	2014-10-05 11:23:02.000000000 +0800
+++ sahara/MANIFEST.in	2014-10-05 11:47:00.000000000 +0800
@@ -37,6 +37,7 @@
 include sahara/tests/unit/resources/*.xml
 include sahara/tests/unit/resources/*.txt
 include sahara/topology/resources/*.xml
+include sahara/plugins/cdh/resources/*.json
 
 exclude .gitignore
 exclude .gitreview
